packages/roosterjs-content-model/lib/editor/corePlugins/ContentModelCopyPastePlugin.ts

```diff
@@ -228,7 +228,16 @@ function selectionExToRange(
     let newRange: Range | null = null;
     if (selection.type === SelectionRangeTypes.TableSelection && selection.coordinates) {
         const table = tempDiv.querySelector(`#${selection.table.id}`) as HTMLTableElement;
-        newRange = createRange(table);
+        const elementAfter = table.parentElement?.insertBefore(
```

Comment:
I think this should be done when we write content model back to DOM, no matter if it is table selection. For example, when handleTable (may need a wrapper handler for table for cut/copy), we directly write a SPAN around TABLE element

Reply:
Then we convert the table selection directly to a regular selection, then here it can select the whole SPAN. (or maybe a DIV will be better)

Reply:
Sounds good.
Updated the PR to change the handleTable behavior to add a div around the table.